File: Java Programs/160905318/week1/NCR.java
import java.util.Scanner;
class NCR{
	int fact(int n){
		int f=1;
		for(int i=2;i<=n;i++)
			f*=i;
		return f;
	}
	public static void main(String[] args){
		Scanner sc=new Scanner(System.in);
		NCR ob=new NCR();
		System.out.println("Enter value of N and R");
		int n=sc.nextInt();
		int r=sc.nextInt();
		int ncr = ob.fact(n)/ob.fact(r)/ob.fact(n-r);
		System.out.println(n+"C"+r+"="+ncr);
	}
}

File: PSUC- C++ Basic Programs/Week 9/nCr.cpp
#include<iostream>
using namespace std;

int fact(int x)
{ int count=1;
  for(int i=x;i>0;i--)
    count=count*i;
  return count;

}

int main()
{ int v,n,r;
 cout<<"Enter value of n and r in nCr ";
 cin>>n>>r;
 v=fact(n)/(fact(n-r)*fact(r));
 cout<<"Value of nCr is "<<v;
 return 0;
}

